### PR TITLE
T18078 utils: drop tests_import

### PR DESCRIPTION
**Pull request by gctucker**
Remove dead code in utils.tests_import which was used with the
previous API.  The new API for imporing test results uses
utils.kci_test instead.  Move code from tests_import to kci_test and
drop the utils.tests_import module.

Signed-off-by: Guillaume Tucker <guillaume.tucker@collabora.com>

### CODE REVIEW

**Comment by gctucker** [comment]
@mgalka Please take a look and confirm this doesn't break the generic API to send test results.

**Comment by mgalka** [comment]
Just one cosmetic change. Technically LGTM.

File: app/utils/kci_test/__init__.py [hunk 1]
```diff
@@ -524,6 +523,43 @@ def _parse_test_group_from_json(test_json, database, errors):
     return test_doc
 
 
+def update_test_group_add_test_case_id(
+        case_id, group_id, group_name, database):
+    """Add the test case ID to the list of a a test group and save it.
```
[the remark this answers is not in the document]
Comment (gctucker) [reply app/utils/kci_test/__init__.py]: Well spotted, fixed now.

**Comment by gctucker** [comment]
Tested OK again on staging.

**Pull request merged**